pages/cloud/dedicated/partitioning_ovh/guide.en-gb.md

```diff
@@ -263,13 +263,13 @@ There are 2 types of errors:
 
 In the next section we will only focus on the **customer errors** types within the **partitioning pre-processing step**, because this is only helpful for the customer.
 
-##### common customer errors
+#### Common customer errors
 
 The following table gives an overview of well known customer errors and how to fix them.
 
 |Error message|Details|Solution(s)|
 |---|---|---|
-|Some Linux distributions such as RHEL family OSes don't support those mountpoints / mountpoint reserved/managed by OVHcloud (\*\*\*). Please remove those mountpoints and restart an installation|- You have chosen `/boot/efi` as mountpoint. OVHcloud will create this partition automatically for you if your server needs one<br />- You have chosen a mountpoint that is symlinked on some operating systems. See [Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/fhs.shtml) for more details.|- Choose another mountpoint for the partition or remove this partition from your partitioning layout|
```

Comment:
@Jessica41 the first column is the exact message that the API returns. We can fix those but I think we should stick to the existing message for now.

Reply:
alright, will revert the changes in the next edit. Thanks! @sbraz